content/en/docs/external-add-ons/kserve/migration.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Migration"
+description = "Migrating from KFServing to KServe"
+weight = 2
+
++++
+
+The migration job will by default delete the leftover KFServing installation after migrating the InferenceServices from
+`serving.kubeflow.org` to `serving.kserve.io`.
+
+### Migrating from Kubeflow-based KFServing
+
+1. Install Kubeflow-based KServe 0.7 using the [install YAML](https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve_kubeflow.yaml)
+    - This will not affect existing services yet.
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kserve/kserve/master/install/v0.7.0/kserve_kubeflow.yaml
+    ```
+
+2. Run the [KServe Migration YAML](https://github.com/kserve/kserve/blob/master/hack/kserve_migration/kserve_migration_job_kubeflow.yaml) for Kubeflow-based installations
+    - This will begin the migration. Any errors here may affect your existing services.
+
+    - If you do not want to delete the KFServing resources after migrating, download and edit the env `REMOVE_KFSERVING`
+      in the YAML before applying it
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kserve/kserve/master/hack/kserve_migration/kserve_migration_job_kubeflow.yaml
+    ```
+
+3. Clean up the migration resources
+
+    ```bash
+    kubectl delete ClusterRoleBinding cluster-migration-rolebinding
+    kubectl delete ClusterRole cluster-migration-role
+    kubectl delete ServiceAccount cluster-migration-svcaccount -n kubeflow
+    ```
+
+4. Update the models web app to use the new InferenceService API group `serving.kserve.io`
+    - Change the deployment image to `kserve/models-web-app:v0.7.0`
+
+    ```bash
+    kubectl edit deployment kfserving-models-web-app -n kubeflow
+    ```
+
+5. Update the cluster role to be able to access the new InferenceService API group `serving.kserve.io`
+    - Edit the `apiGroups` from `serving.kubeflow.org` to `serving.kserve.io`
+    - This is a temporary fix until the next Kubeflow release includes these changes
+
+    ```bash
+    kubectl edit clusterrole kfserving-models-web-app-cluster-role
+    ```
```

Comment:
@kimwnasptd Should this work out of the box now with web app 0.7 release?

Reply:
The new 0.7 manifests have an overlay for working with a Kubeflow installation out of the box. This means to migrate, for the web app, someone would need to:
1. Apply the new 0.7 manifests of the web app
2. Run some kubectl commands we can provide in these docs to uninstall the previous parts
3. Edit the KF CentralDashboard to use the KServe app's endpoint [`/kserve-endpoints`]

But I'd agree with @pvaneck on doing this on a follow up PR, to try and slightly reduce the size of this PR https://github.com/kubeflow/website/pull/3162#issuecomment-1061075612